.circleci/config.yml

```diff
@@ -96,9 +96,7 @@ jobs:
       - run:
           name:  Run E2E test
           command: ./test/tools/integration/run_e2e_tests.sh
-      - run:
-          name: Test create and destroy centos machine
-          command: ./test/tools/integration/create-and-destroy-machine.sh centos
+          no_output_timeout: 15m
```

Comment:
I have increased this timeout to be greater than the running time of the tests so that we can grab the logs.